src/core/a-entity.js

```diff
@@ -241,21 +240,11 @@ var proto = Object.create(ANode.prototype, {
 
       if (this.hasLoaded) { return; }
 
-      // Attach to parent object3D.
-      this.addToParent();
-
-      // Scene load.
-      function sceneLoadCallback () { self.updateComponents(); }
-      if (this.isScene) {
-        ANode.prototype.load.call(this, sceneLoadCallback);
```

Comment:
All entities are also nodes. Do we need to specify what each waits for? Both scene and entities wait for all the children nodes to load.